app/javascript/components/common/skeleton/skeletons/ReputationDropdownSkeleton.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { SkeletonShape } from '../components/SkeletonShape'
+import { SkeletonLoader } from '../components/SkeletonLoader'
+
+export function ReputationDropdownSkeleton() {
+  return (
+    <SkeletonLoader>
+      <SkeletonShape
+        shape="tag"
+        style={{
+          height: '38px',
+          width: '100px',
```

Comment:
Rather than hardcoding this, how about we set the width to be `x+(nd*s)` where x is the fixed width of the left/right, `nd` is the number of digits and `s` is a fixed "space per digit" thing (e.g. 10px). We can set the same on the real component. I know it won't be perfectly correct as the characters aren't the right width, but if we just bleed the extra space into the left/right padding, by centralising the content, I don't think it'll matter.